cni/pkg/plugin/testdata/dns.txt.golden

```diff
@@ -11,30 +12,33 @@
 -A ISTIO_INBOUND -p tcp --dport 15021 -j RETURN
 -A ISTIO_INBOUND -p tcp --dport 15090 -j RETURN
 -A ISTIO_INBOUND -p tcp -j ISTIO_IN_REDIRECT
--A OUTPUT -p tcp -j ISTIO_OUTPUT
+-A OUTPUT -j ISTIO_OUTPUT
 -A ISTIO_OUTPUT -p tcp --dport 15020 -j RETURN
+-A ISTIO_OUTPUT -p udp --dport 15020 -j RETURN
 -A ISTIO_OUTPUT -o lo -s 127.0.0.6/32 -j RETURN
 -A ISTIO_OUTPUT -o lo ! -d 127.0.0.1/32 -p tcp -m multiport ! --dports 53,15008 -m owner --uid-owner 1337 -j ISTIO_IN_REDIRECT
 -A ISTIO_OUTPUT -o lo -p tcp ! --dport 53 -m owner ! --uid-owner 1337 -j RETURN
 -A ISTIO_OUTPUT -m owner --uid-owner 1337 -j RETURN
 -A ISTIO_OUTPUT -o lo ! -d 127.0.0.1/32 -p tcp ! --dport 15008 -m owner --gid-owner 1337 -j ISTIO_IN_REDIRECT
 -A ISTIO_OUTPUT -o lo -p tcp ! --dport 53 -m owner ! --gid-owner 1337 -j RETURN
 -A ISTIO_OUTPUT -m owner --gid-owner 1337 -j RETURN
--A ISTIO_OUTPUT -p tcp --dport 53 -j REDIRECT --to-ports 15053
+-A ISTIO_OUTPUT -j ISTIO_OUTPUT_DNS
+-A ISTIO_OUTPUT_DNS -p tcp --dport 53 -j REDIRECT --to-ports 15053
+-A ISTIO_OUTPUT_DNS -p udp --dport 53 -m owner --uid-owner 1337 -j RETURN
```

Comment:
A bit confused. Why do we need this rule here, isn't it handled in the protocol-agnostic checks (L21 and L24 of the golden file)?